app/services/parse_website_for_data.rb

```diff
@@ -0,0 +1,80 @@
+class ParseWebsiteForData
+  WEBSITE = "https://www.greenvillesc.gov".freeze
+  PAGE = "/375/Business-License-Reports".freeze
+  LINKS_DIV = ".fr-view ul li a".freeze
```

Comment:
Change this line if you'll need to parse another css block (another block with links on the page)